awxkit/awxkit/api/pages/workflow_job_templates.py

```diff
@@ -15,6 +15,27 @@ class WorkflowJobTemplate(HasCopy, HasCreate, HasNotifications, HasSurvey, Unifi
     optional_dependencies = [Organization]
     NATURAL_KEY = ('organization', 'name')
 
+    @property
+    def related(self):
```

Comment:
I'm pretty uncomfortable with doing it this way.  However, fortunately, I do note that the `inventory` field shows up as of type `'id'` in the `OPTIONS` call on the api endpoint even under current devel awx.  I think it would be easy enough to check for the case of an id field that fails to show up in related in the `_export()` method, and attempt to construct the natural key by other means (probably by doing a `_filtered_list()` call).

Reply:
OK. Good suggestion. I'll look into it and push an update if I get something working.